sencha-workspace/SlateTasksStudent/app/view/TaskTree.js

```diff
@@ -216,16 +216,33 @@ Ext.define('SlateTasksStudent.view.TaskTree', {
         }
     },
 
+    afterTasksLoad: function() {
+        var tree = this,
+            sublists = tree.el.select('.slate-tasktree-sublist');
+
+        sublists.each(function(sublist) {
```

Comment:
@ryon to avoid any race conditions, update the top of this function like this:

```javascript
var tree = this;

if (!tree.rendered) {
    tree.on('render', 'afterTasksLoad', this, { single: true} });
    return;
}

tree.el.select('.slate-tasktree-sublist').each(function(sublistEl) {
    // ...
});
```